src/webgl/p5.RendererGL.js

```diff
@@ -178,11 +175,16 @@ p5.RendererGL.prototype._resetContext = function(attr, options, callback) {
     document.body.appendChild(c);
   }
   this._pInst.canvas = c;
+
+  this._pInst.push();
```

Comment:
_resetContext sounds to me like it would re-establish renderer defaults, but they look like they're being preserved here. it's worth a comment here to clarify the method's intent.

Reply:
yeah, i agree. the reason i did this was that calling `setAttributes` calls `_resetContext` (actually it's the _only_ thing that calls _resetContext) which means that

    createCanvas(100, 100, WEBGL);
    fill(150);
    setAttributes('antialias', true);

results in the effects of the `fill` call being lost.

maybe we can move the push/pop into `setAttributes()` ?

Reply:
that seems good!

Reply:
just noting that it seems this hasn't been addressed yet.

Reply:
oops, i missed this one.